app/Providers/AppServiceProvider.php

```diff
@@ -30,5 +32,19 @@ public function boot()
         /** @var MailSettings $settings */
         $settings = $this->app->make(MailSettings::class);
         $settings->setConfig();
+
+
+        //get the current Branch name and latest git Version the Instance is running on and safe it as a config variable
+        try {
```

Comment:
maybe this could be refactored to a helper class, to keep the app service provider clean, *just a note*

Reply:
i really dont know how to put this into a helper function to make actual sense....maybe you want to look into that but i couldnt find a satisfying way to do it really @AVMG20

Reply:
i can merge it later and try and refactor it my self :)